### ISSUE TITLE
[FOR FANS], we can make the donation: COATOOL for alive, continbue..

### ISSUE THREAD

**Issue by manuelsongokuh** [ndee85/coa_tools#194]
hi

this COATOOLS is important!! so i ask all you , who wants/likes this coatools so we need to help to @ndee85 for future alive this coatools, new versions..

but we (fans,clients,artists,etc) need to organize for donation to @ndee85  developing coatools..

coatools for blender needs a lot of time and energy of @ndee85 so i think  " correctly " to gift the donation..

but we (fans..) need  find one way for donation.. example:
patreon but maybe @ndee85 will be could open new project in patreon for to continue developing for COATOOLS..
 
i ask to all fans please...make " gather " for donation...

i write myself (my think) ,

**Comment by ktorenz** [comment]
I suggest @ndee85 you need to place your addon in the Blender Market. Advertise it to the community and sell it. I think you could profit that way.

**Comment by 3dcinetv** [comment]
I will buy COA Tools. I need this for my every day workflow. I would like to continue to use it in Blender 3.5 and beyond.
Subscribing to this petition thread.

**Comment by manuelsongokuh** [comment]
good idea!, for to keep alive this COA Tools developing..

i hope @ndee85 will accept to @3dcinetv to be continue developing way to keep alive..

**Comment by manuelsongokuh** [comment]
@3dcinetv please advise here so i will follow news,updates,commit..thank you

**Comment by SuitableForWork** [comment]
Yes, please put this on BlenderMarket, Gumroad anywhere that motivates you to continue, and we will pay if it's going to COA tools!
